SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,809 @@
+  </header-bar-section>
+   
+   
+  <!-- HeroSection Example -->
+  <!-- About Section -->
+  	
+  <!-- Hero Section -->
+  <section class="Hero-section py-5">
+    <div class="container">
+      <h1 class="text-center mb-4">Our Culinary Journey</h1>
+      <p class="lead text-center">
+       <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit.</br>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </br> ]

(tag-pair)